src/garage/sampler/is_sampler.py

```diff
@@ -172,11 +173,7 @@ def sample_isweighted_paths(
             path['rewards'] *= is_ratio
 
         if ess_threshold:
-            if kong_ess(is_weights) < ess_threshold:
+            if len(is_weights) / (1 + var(is_weights)) < ess_threshold:
```

Comment:
if you want to eliminate the helper function, please leave a comment that this is the effective sample size estimate used by Kong 1992:

```
Kong, Augustine. "A note on importance sampling using standardized weights." University of Chicago, Dept. of Statistics, Tech. Rep 348 (1992).
```